tensorflow/python/training/learning_rate_decay.py

```diff
@@ -19,7 +19,8 @@
 from __future__ import print_function
 
 from tensorflow.python.framework import ops
-from tensorflow.python.ops import math_ops
+from tensorflow.python.ops.math_ops import add, cast, div, exp, floor, mod, mul, neg
```

Comment:
@vrv Thanks Vijay for your comments.  I see, I just thought it makes the equation easier to read.  ie. Line 137 exponent = exp(mul(neg(decay_rate), epoch)).  I'll change it and resubmit.